pkg/webhooks/leaderworkerset_webhook.go

```diff
@@ -97,6 +102,7 @@ func (r *LeaderWorkerSetWebhook) ValidateUpdate(ctx context.Context, oldObj, new
 	if newLws.Spec.LeaderWorkerTemplate.SubGroupPolicy == nil && oldLws.Spec.LeaderWorkerTemplate.SubGroupPolicy != nil {
 		allErrs = append(allErrs, field.Invalid(specPath.Child("leaderWorkerTemplate", "SubGroupPolicy", "subGroupSize"), oldLws.Spec.LeaderWorkerTemplate.SubGroupPolicy.SubGroupSize, "cannot remove subGroupSize after enabled"))
 	}
+
```

Comment:
we should validate that lws.Spec.NetworkConfig.SubdomainPolicy can not be updated to nil

Reply:
If we default `subdomainPolicy` in the case where `networkConfig` is set but not `subdomainPolicy`, doesn't that cover the case where it is updated to nil? So it is valid to update it to nil, it would just be equivalent to setting it to `Shared`.